packages/query/src/index.ts

```diff
@@ -460,7 +468,11 @@ const generateQueryOptions = ({
 
   return `${
     !isObject(options) || !options.hasOwnProperty('enabled')
-      ? `enabled: !!(${params.map(({ name }) => name).join(' && ')}),`
+      ? isVue(outputClient)
+        ? `enabled: computed(() => !!unref(${params
```

Comment:
@Maxim-Mazurok  - Should version being `0` here be also covered? Thanks!

Reply:
Yeah... Ideally IMO we should probably check each argument for `x !== null && x !== undefined`. But that would probably require a major version bump as it might break other people's code...
I think this is probably good enough for now, maybe we can create a separate issue to handle falsy values.

Reply:
There's a related TS issue: https://github.com/anymaniax/orval/issues/858